app/models/qdm/participation.rb

```diff
@@ -4,8 +4,7 @@ class Participation < DataElement
     include Mongoid::Document
     embedded_in :patient
     field :participationPeriod, type: QDM::Interval
-    field :hqmfOid, type: String, default: '2.16.840.1.113883.10.20.28.4.130'
```

Comment:
Similar comment above about this being the outlier

Reply:
See other comment on participation.